tests/expected/safety-constraint-attribute/abstract-value/abstract-value.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! Check that the `#[safety_constraint(...)]` attribute automatically generates
```

Comment:
This comment is no longer valid (the `Arbitrary` implementation is no longer automatically derived)?

Reply:
True, thanks! Updated the comment.